RoadRangerBackEnd/Report/MailReportSender.cs

```diff
@@ -27,14 +28,21 @@ public void SendGeneretedMail(Car car)
 
         public void SendMail(string subject, string body)
         {
-			MailMessage mail = new MailMessage
-			{
-				From = mailAddress    //Mūsų gmail laikinas, iš kurio siunčia
-			};
-			mail.To.Add(defaultRecipient);     //adresas iš textbox
-            mail.Subject = subject;     //Tema
-            mail.Body = body;           //Tekstas
-            smtpServer.Send(mail);      //siunčiam
+            try
+            {
+                MailMessage mail = new MailMessage
+                {
+                    From = mailAddress    //Mūsų gmail laikinas, iš kurio siunčia
+                };
+                mail.To.Add(defaultRecipient);     //adresas iš textbox
+                mail.Subject = subject;     //Tema
+                mail.Body = body;           //Tekstas
+                smtpServer.Send(mail);      //siunčiam
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
```

Comment:
Maybe show the message in the app with some kind of pop up would be better than Console logging it